demos/button.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+  -->
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>MDL Button Demo</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
```

Comment:
This doesn't load weight 500, which is what you're using, and so the text looks wrong unless you already have Roboto on your system. Please use something like `https://fonts.googleapis.com/css?family=Roboto:300,400,500`.